recipes-omnect/initrdscripts/omnect-os-initramfs/flash-mode-1

```diff
@@ -7,25 +7,25 @@
 # Flash Mode
 #     1    clone disk image from active block device to other one
 #
-OMNECT_FLASH_MODE=""
+FLASH_MODE=""
 
-OMNECT_FLASH_MODE_VAR="flash-mode"
-OMNECT_FLASH_DEVPATH_VAR="flash-mode-devpath"
+FLASH_MODE_VAR="flash-mode"
+FLASH_DEVPATH_VAR="flash-mode-devpath"
```

Comment:
wouldn't FLASH_MODE_DEVPATH_VAR more consistent in naming here? (yes, not only now but right from the start ;-))
at least it's also a variable dealing with paramters passed on by user like FLASH_MODE_IP_ADDR.

Reply:
done